src/components/FormManager.jsx

```diff
@@ -42,6 +42,16 @@ class FormManager extends React.Component {
     return this.context.getFormQuestionsData();
   };
 
+  printFormSpecification = () => {
```

Comment:
I would rather have a method to return string than print to console ... maybe I suggested the name incorrectly

User can always do console.log(printFormSpecification).

Maybe `getFormSpecification` ?